packages/react/src/lib/renderComponent.tsx

```diff
@@ -203,7 +195,7 @@ const renderComponent = <P extends {}>(
     displayName,
     props: stateAndProps,
     variables: resolvedVariables,
-    theme: context.theme,
```

Comment:
`context` can be `undefined` if there are no wrapping `Provider` and it will throw an error